x/gravity/keeper/attestation.go

```diff
@@ -27,7 +28,7 @@ func (k Keeper) Attest(
 		return nil, fmt.Errorf("could not find ValAddr for delegate key, should be checked by now")
 	}
 	valAddr := val.GetOperator()
-	if err := sdk.VerifyAddressFormat(valAddr); err != nil {
+	if err := sdk.VerifyAddressFormat([]byte(valAddr)); err != nil {
```

Comment:
What about using `valAddr.Bytes()`?

Reply:
as valAddr is a string, it's not possible to do it that way

Reply:
This is not the correct way to convert Bech32 string to bytes. Use address codec for converting bech32 to bytes.